src/d3/scatterplot.d3.ts

```diff
@@ -124,9 +125,11 @@ export class Scatterplot{
   }
 
   private generateText(){
+    this.scalePosition();
     // enter identifies any DOM elements to be added when # array elements doesn't match
     d3.select(this.dataTextContainer).selectAll('a-entity').data(this.data).enter().append('a-entity');
-    d3.select(this.dataTextContainer).selectAll('a-entity')
+    this.cardSelection =  d3.select(this.dataTextContainer).selectAll('a-entity');
```

Comment:
Do you need the variable? E.g.:

```
d3.select(this.dataTextContainer).selectAll('a-entity')
  .attr('geometry', 'primitive: plane; height: auto; width: .5')
  .attr(...);
...

Reply:
I end up needing this to filter through when the user is hovering over a sphere and wants to look at a card... I add a `mouseenter` event listener to the sphere that will use `cardSelection`